doc/man/8/ceph.rst

```diff
@@ -320,6 +322,16 @@ Example::
 	ceph daemon osd.0 help
 
 
+daemonhistogram
+---------------
+
+Visualize Ceph daemons' performance histogram.
+
+Usage::
+
+	ceph daemonperf {type.all|daemon_names|socket_paths} [ {batch|live} [{interval} [{count}]]]
```

Comment:
can this describe the meaning of the parameters / that you need to run it on the same host as the daemon?